src/test/java/net/sf/jabref/importer/fileformat/BibteXMLImporterTest.java

```diff
@@ -0,0 +1,185 @@
+    public void testGetFormatName() {
+        BibteXMLImporter importer = new BibteXMLImporter();
+        Assert.assertEquals("BibTeXML", importer.getFormatName());
+    }
+
+    @Test
+    public void testGetCLIId() {
+        BibteXMLImporter importer = new BibteXMLImporter();
+        Assert.assertEquals("bibtexml", importer.getCLIId());
+    }
+
+    @Test
+    public void testIsRecognizedFormatAccept() throws IOException {
+        BibteXMLImporter importer = new BibteXMLImporter();
+
+        List<String> list = Arrays.asList("BibteXMLImporter.xml", "BibteXMLImporterEmpty.xml");
+
+        for (String str : list) {
+            try (InputStream is = BibteXMLImporter.class.getResourceAsStream(str)) {
+                Assert.assertTrue(importer.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testIsRecognizedFormatReject() throws IOException {
+        BibteXMLImporter importer = new BibteXMLImporter();
+
+        List<String> list = Arrays.asList("oai2.xml", "oai22.xml", "oai23.xml");
+
+        for (String str : list) {
+            try (InputStream is = BibteXMLImporter.class.getResourceAsStream(str)) {
+                Assert.assertFalse(importer.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testImportEntries() throws IOException {
+
+        BibteXMLImporter importer = new BibteXMLImporter();
+
+        try (InputStream is = BibteXMLImporter.class.getResourceAsStream("BibteXMLImporter.xml")) {
+            List<BibEntry> entries = importer.importEntries(is, new OutputPrinterToNull());
+
+            BibEntry entry = entries.get(0);
+            Assert.assertEquals(entry.getCiteKey(), "");
+            Assert.assertEquals("article", entry.getType());
+            Assert.assertEquals("Test Article", entry.getField("article"));
+
+            entry = entries.get(1);
+            Assert.assertEquals(entry.getCiteKey(), "");
+            Assert.assertEquals("inbook", entry.getType());
+            Assert.assertEquals("Test InBook", entry.getField("inbook"));
+
+            entry = entries.get(2);
+            Assert.assertEquals(entry.getCiteKey(), "");
+            Assert.assertEquals("book", entry.getType());
+            Assert.assertEquals("Test Book", entry.getField("book"));
+
+            entry = entries.get(3);
+            Assert.assertEquals("book", entry.getType());
+            Assert.assertEquals("musterman", entry.getCiteKey());
+            Assert.assertEquals("musterman", entry.getField("bibtexkey"));
+            Assert.assertEquals("Musterman, M.", entry.getField("author"));
+            Assert.assertEquals("Java is good!", entry.getField("title"));
+            Assert.assertEquals("Book publisher", entry.getField("publisher"));
+            Assert.assertEquals("2016", entry.getField("year"));
+            Assert.assertEquals("Stuttgart", entry.getField("address"));
+            Assert.assertEquals("Muster, M.", entry.getField("person"));
```

Comment:
I searched for all available fields in the type Book. There was this field person. So I took it to test whether JabRef can read the value of this type.
I guess a correct use is:
```xml
    <bibtex:editor>
      <bibtex:person>Mustermann, M.</bibtex:person>
    </bibtex:editor>
```
In this and the following way JabRef wouldn´t have found the value of the field "editor".
```xml
    <bibtex:editor>Muster, M.</bibtex:editor>
    <bibtex:editor>
      <bibtex:person>Mustermann, M.</bibtex:person>
    </bibtex:editor>
```
Only if I would use this format JabRef would read both values correctly.
```xml
    <bibtex:editor>
      <bibtex:person>Mustermann, M.</bibtex:person>
    </bibtex:editor>
    <bibtex:editor>Muster, M.</bibtex:editor>
```
So my goal was only to get the values of the available fields. This way there is less code and a better overview over the field values.

To answer your question. If the first code example is correct it´s a way to add persons to the group authors or editors. But when I tested it, only the last entry is returned. And I can only reach the field person. There is no "editors" field to get a list of all person who are the editors.
If I am wrong please correct me.